curses.gemspec

```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new { |s|
   s.name = "curses"
-  s.version = "1.2.4"
+  s.version = "1.2.5"
```

Comment:
Please leave this unchanged in this PR.  It's all up to maintainers when and how much to bump the version number.

Reply:
Whoops, you're right. Used this to distinguish my local install 🙇‍♂️